pkg/middleware/operation_complexity/operation_complexity.go

```diff
@@ -27,6 +27,21 @@ import (
 	"github.com/jensneuse/graphql-go-tools/pkg/operationreport"
 )
 
+type GlobalComplexityResult struct {
```

Comment:
what if make this type a generic container of this 3 fields, like 
```
type ComplexityResults struct {
	NodeCount  int
	Complexity int
	Depth      int
}

type FieldComplexityResult struct {
	TypeName   string
	FieldName  string
	Alias      string
	ComplexityResult
}
```

And in graphql package it also could be used
```
type ComplexityResult struct {
	operation_complexity.ComplexityResult
	PerRootField []FieldComplexityResult
	Errors       Errors
}
```

And maybe in this package struct name could be more meaningful like `OperationStats`